apps/payments/next/app/[locale]/[offeringId]/checkout/[interval]/[cartId]/success/en.ftl

```diff
@@ -1,7 +1,7 @@
 next-payment-confirmation-thanks-heading-account-exists = Thanks, now check your email!
 
 # $email (String) - The user's email.
-next-payment-confirmation-thanks-subheading = You’ll receive an email at { $email } with instructions about your subscription, as well as your payment details.
+payment-confirmation-thanks-subheading-account-exists-2 = You’ll receive an email at { $email } with instructions about your subscription, as well as your payment details.
```

Comment:
Just adding a new fluent id for this new string
- `payment-confirmation-thanks-subheading` belonged to the string that began `A confirmation email has been sent to...`.
- as @david1alvarez previously pointed out it's a new string that differs from another one that was previously used (`payment-confirmation-thanks-subheading-account-exists - You’ll receive an email at { $email } with instructions for setting up your account, as well as your payment details.`)

cc: @mozilla/fxa-l10n